CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- Add grafanaOrganization CRD in helm chart.
```

Comment:
I don't see any changes in the helm chart. Is there some magic I'm missing?

Reply:
My bad I removed it due to conflicts and forgot to add the symlink back

Reply:
There is a symlink to the kubernetes CRD file in the `config/crd` directory that was unintentionally removed but is now back there :)

Reply:
I see it now! :+1: